lib/xml/parser.js

```diff
@@ -84,7 +84,7 @@ function parseEvents(xml) {
 
             etag = etag.substring(1, etag.length);
 
-            const data = {};
+            let data = {};
```

Comment:
Please never use a variable with `data` as naming. Somehow everything is data. Please be a lot more specific.